<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTTREdit</class>
 <widget class="QWidget" name="FormTTREdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1,1000">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="System">
      <attribute name="title">
       <string>System</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sound system (for playing)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbmSystem"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Order and Patterns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
         <item>
          <widget class="QGroupBox" name="grpOrder">
           <property name="title">
            <string>Order</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
              <item row="2" column="1">
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <widget class="QPushButton" name="btnNewOrder">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="btnDeleteOrder">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="btnOrderMoveUp">
                  <property name="text">
                   <string>Move up</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="btnOrderMoveDown">
                  <property name="text">
                   <string>Move down</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QListWidget" name="lstOrders">
                <property name="font">
                 <font>
                  <family>Courier New</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpPatterns">
           <property name="title">
            <string>Patterns</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="2">
               <widget class="QPushButton" name="btnDeletePattern">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QComboBox" name="cmbPatterns"/>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="btnNewPattern">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Instruments</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btnNewOrder_2">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Presets</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btnDeleteOrder_2">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cmbPredefinedInstrument"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Instrument:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="cmbInstruments"/>
       </item>
       <item row="2" column="2" colspan="3">
        <widget class="QPushButton" name="btnSetInstrument">
         <property name="text">
          <string>Set instrument from preset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="3">
        <widget class="QLineEdit" name="leInstrument"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="leInstrumentData"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>TRSE player instrument:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="cmbTRSEInstrument"/>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnTRSEInstruments">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="leText">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnPlay">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="lyTrackers"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
